qt-webkit@2.3.rb

```diff
@@ -0,0 +1,30 @@
+class QtWebkitAT23 < Formula
+  desc "Qt port of WebKit (insecure, you shouldn't use it)"
```

Comment:
I've updated the warnings to make them more clear. In my opinion, the majority of `qt-webkit@2.3`'s uses (help files, content provided by the user in e.g. a trusted KML document) are perfectly fine -- it's just untrusted third-party Web content that should be avoided.